src/features/user/Account/components/AccountRecord.tsx

```diff
@@ -167,7 +168,10 @@ export function DetailsComponent({ record }: DetailProps): ReactElement {
         <div className={styles.singleDetail}>
           <p className={styles.title}>{tMe('project')}</p>
           {record.projectGuid ? (
-            <a title={record.details.project} href={`/${record.projectGuid}`}>
+            <a
+              title={record.details.project}
+              href={`/prd/${record.projectGuid}?backNavigationUrl=${router.asPath}`}
```

Comment:
I feel we could create a util function to generate project path in a standard way for each place.